elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/ElasticsearchConnectionSettings.scala

```diff
@@ -4,10 +4,15 @@
 
 package akka.stream.alpakka.elasticsearch
 
+import akka.http.scaladsl.HttpsConnectionContext
+import akka.http.scaladsl.model.HttpHeader
+
 final class ElasticsearchConnectionSettings private (
     val baseUrl: String,
     val username: Option[String],
-    val password: Option[String]
+    val password: Option[String],
+    val headers: Option[List[HttpHeader]],
```

Comment:
Could probably be initialized as an empty list.

Reply:
Changed to an initialized empty list instead of an option.